autorest/azure/example/main.go

```diff
@@ -17,83 +18,128 @@ import (
 
 const resourceGroupURLTemplate = "https://management.azure.com/subscriptions/{subscription-id}/resourcegroups"
 const apiVersion = "2015-01-01"
+const xplatClientID = "04b07795-8ddb-461a-bbee-02f9e1bf7b46"
 
```

Comment:
Yes. It's the xplat client id. I debated whether or not to leave it in. It's makes the example easier to use, but I'm fine with removing it.